styles/email.scss

```diff
@@ -0,0 +1,31 @@
+//
+//  Styles for emails
+//  _____________________________________________
+
+//  See comments in source/_email-base.less for explanation of email styles
+
+@import 'vendor/magento-ui/lib'; // Global lib
+@import 'vendor/magento-ui/variables/email'; // Global email variables
+@import 'variables'; // Local theme variables
+@import 'email/email-variables'; // Theme variables for emails
+
+// Mixins
+@import "mixins/media-queries";
+
+//  Include all styles that have been indicated as non-inline styles
+@import 'email/email-base';
+@import 'email/email-extend';
+
+//
+//  Module-specific styles for emails
+//  TODO: decide if we will update this list manually or we will add empty _email.scss to each Module_Namespace/styles directory
```

Comment:
Is this our comment or it's inherited from the core?

Reply:
added. removed, left  line 20.